src/python/pants/backend/python/subsystems/lambdex.py

```diff
@@ -31,43 +30,30 @@ class Lambdex(PythonToolBase):
     lockfile_rules_type = LockfileRules.SIMPLE
 
     layout = EnumOption(
-        default=LambdexLayout.LAMBDEX,
+        default=LambdexLayout.ZIP,
         help=softwrap(
             """
             Explicitly control the layout used for `python_awslambda` and
             `python_google_cloud_function` targets. This option exists for the transition from
             Lambdex-based layout to the plain zip layout, as recommended by cloud vendors.
             """
         ),
+        removal_version="2.19.0.dev0",
+        removal_hint=softwrap(
+            """
+            Remove the whole [lambdex] section, as Lambdex is deprecated and its functionality be
```

Comment:
```suggestion
            Remove the whole [lambdex] section, as Lambdex is deprecated and its functionality will be
```

Reply:
Added to https://github.com/pantsbuild/pants/issues/19067 as a reminder.